pom.xml

```diff
@@ -15,6 +15,10 @@
     </repository>
   </repositories>
 
+  <properties>
+    <jenkins.version>1.642.3</jenkins.version>
```

Comment:
I think that's OK, we can bump things, but we need to document the minimum new requirement clearly in CHANGELOG and README and probably increment a major version of the plugin in this PR.